extensions/amp-access/0.1/test/test-amp-access-client.js

```diff
@@ -15,20 +15,25 @@
  */
 
 import {AccessClientAdapter} from '../amp-access-client';
+import * as lolex from 'lolex';
 import * as sinon from 'sinon';
 import * as mode from '../../../../src/mode';
 
-describe('AccessClientAdapter', () => {
 
-  let sandbox;
+describes.realWin('AccessClientAdapter', {
+  amp: true,
+}, env => {
+  let win;
+  let ampdoc;
   let clock;
   let validConfig;
   let context;
   let contextMock;
 
   beforeEach(() => {
-    sandbox = sinon.sandbox.create();
-    clock = sandbox.useFakeTimers();
+    win = env.win;
+    ampdoc = env.ampdoc;
+    clock = lolex.install(win);
```

Comment:
Unfortunately, sinon's version (very annoyingly so) does not allow installing timer on a specified window, only global one.